### ISSUE TITLE
Built with older ncurses than found on stable distros

### ISSUE THREAD

**Issue by tsloughter** [wojtekmach/otp_releases#1]
I tried using the 24-rc1 build and it fails on my system because it wants `libtinfo.so.5` but on Fedora 33 I have `libtinfo.so.6`.

I think .6 has been used for years, are these being built on Ubuntu 18 or 16?

**Comment by wojtekmach** [comment]
I’m building it on Ubuntu 16.04, here’s an example job (for 23.3.1): https://github.com/wojtekmach/beamup/actions/runs/700830788

**Comment by tsloughter** [comment]
Oh, my bad, I was looking for it in here.

16 is end of life this month, will you be moving to 18 soon? I'm opening a Build and Packaging agenda item in its repo to discuss how to version the packages. I'm happy to send a PR to the build script to run against more distros, but first need to know how we should name these packages so that a tool like erlup can check the local system it is running on and map that to the package to download.

I think we discussed in the meetings before about having at least the Ubuntu version in the package name, then I can have `erlup` check the local version of `libtinfo` and know if it needs the packages built on 16 or 18 or X.

Closing this issue so discussion moves to the WG.
